### ISSUE TITLE
Returning results as dictionary silently throws away tags for words that appear > once

### ISSUE THREAD

**Issue by jjjamie** [kamalkraj/BERT-NER#14]
https://github.com/kamalkraj/BERT-NER/blob/c12b2ecf2b7572610d75184e7eed02a778645ce3/bert.py#L94

Have made it work for me locally with:
`output = [ {"word": word, "tag": label, "confidence": confidence} for word, label, confidence in zip(words, labels, logits_confidence) ]`

**Comment by kamalkraj** [comment]
@jjjamie Thanks
fixed 41682f78651c0f1dcf787f5e59e1b9ca99ee97e5
